worlds/rabi_ribi/existing_randomizer/.gitignore

```diff
@@ -7,3 +7,4 @@ converter/*.txt
 converter/generated/
 converter/modified/
 converter/original/
+!minimap_raw.png
```

Comment:
Was this supposed to be gitignored? Its included in the commit.

Reply:
it's included in the standalone for use in the visualizer; I can just remove it for now though (I doubt we'd build a tracker in the client)